Doc/howto/descriptor.rst

```diff
@@ -1366,6 +1366,10 @@ Using the non-data descriptor protocol, a pure Python version of
         def __call__(self, *args, **kwds):
             return self.f(*args, **kwds)
 
+        @property
+        def __annotations__(self):
+            return self.f.__annotations__
```

Comment:
You need to delete the reference to `__annotations__` in the following paragraph if you want to make this change